mii/models/load_models.py

```diff
@@ -64,44 +58,42 @@ def load_models(task_name,
                                                  mii_config)
         inf_config["training_mp_size"] = 2
         inf_config["config"] = inference_pipeline.neox_args
+        """
     elif provider == mii.constants.ModelProvider.DIFFUSERS:
         from mii.models.providers.diffusers import diffusers_provider
-        inference_pipeline = diffusers_provider(model_path,
-                                                model_name,
-                                                task_name,
-                                                mii_config)
-        inf_config["replace_with_kernel_inject"] = False  #not supported yet
+
+        inference_pipeline = diffusers_provider(deployment_config)
         inf_config["enable_cuda_graph"] = True
     else:
         raise ValueError(f"Unknown model provider {provider}")
-
+    """
```

Comment:
Need to re-enable this print. I think it's important to give these snippets of output for users to understand the progress of MII deployment (and see what optimizations are being applies). However, I'm wondering if this should be moved elsewhere, maybe a utils function that takes in mii_config or deployment_config and prints useful things?